core/create/src/index.ts

```diff
@@ -44,9 +44,19 @@ function installHooks(logger: typeof winstonLogger) {
   }).default(logger)
 }
 
-function hasToolKitConflicts(error: unknown) {
-  // we need to import hasToolkitConflicts from the app itself instead of npx or else hasToolkitConflicts and ToolKitConflictError will come from npx but error will come from the app level, leading to error failing the instanceof ToolKitConflictError check
-  return (importCwd('@dotcom-tool-kit/error') as typeof ToolkitErrorModule).hasToolKitConflicts(error)
+function hasToolKitConflicts(error: unknown): boolean {
+  try {
+    // we need to import hasToolkitConflicts from the app itself instead of npx
+    // or else hasToolkitConflicts and ToolKitConflictError will come from npx
+    // but the error will come from the app level, leading to the error failing
+    // the instanceof ToolKitConflictError check
+    const errorModule = importCwd('@dotcom-tool-kit/error') as typeof ToolkitErrorModule
+    return errorModule.hasToolKitConflicts(error)
+  } catch {
+    // if the error package isn't found then that probably means npm install
+    // has failed and we should be logging out that error
+    return false
```

Comment:
**question:** This function is just to determine how to log an error in a catch block therefore returning `false` here will still log the error. Have I understood this correctly?

Reply:
[This block[(https://github.com/Financial-Times/dotcom-tool-kit/blob/f67f46fb83074ffa602cbfa6e2572cd424a7f90c/core/create/src/index.ts#L86-L92) is where that function is called so returning false will run the `else` block which will log the actual error rather than the `finished installing hooks, but found conflicts` error:

```js
if (hasToolKitConflicts(error)) {
      logger.log(id, { status: 'done', message: 'finished installing hooks, but found conflicts' })
    } else {
      logger.log(id, {
        status: 'fail',
        message: labels.fail,
        error: loggerError.logged ? undefined : loggerError
      })
    }
```